collections/_article/release-candidate-godot-4-1-rc-1.md

```diff
@@ -0,0 +1,93 @@
+
+-----
+
+*The illustration picture for this article is from* [**Katana Dragon**](https://twitter.com/KatanaDragon_) *— an action RPG developed by [Tsunoa Games](https://twitter.com/tsunoagames) with Godot 4. When the game releases you will be able to venture into a charming voxel world to explore dungeons, avoid traps, and solve puzzles! Follow the development of the game on [Twitter](https://twitter.com/KatanaDragon_) so you don't miss the release date.*
+
+## What's new
+
+41 contributor submitted around 70 improvements for this release. You can review the complete list of changes with our [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.1-rc1), which contains links to relevant commits and PRs for this and every previous release.
+
+There is a number of major changes in Godot 4.1, and you can read more about them in our [earlier announcement](/article/dev-snapshot-godot-4-1-beta-1/). Below are the most notable changes compared to 4.1 beta 3:
+
+- 2D: Tilemap: Update indices after removing custom data layers ([GH-78492](https://github.com/godotengine/godot/pull/78492)).
+- 2D: Tilemap: Fix tile resizing towards atlas boundary ([GH-76152](https://github.com/godotengine/godot/pull/76152)).
+- 2D: Fix click-selecting Sprites with repeated texture ([GH-78566](https://github.com/godotengine/godot/pull/78566)).
+- C#: Fix editor integration breaking and causing error spam when reloading assemblies fails ([GH-75533](https://github.com/godotengine/godot/pull/75533)).
+- C#: Fix condition blocking .NET project build ([GH-78488](https://github.com/godotengine/godot/pull/78488)).
+- Core: Ensure `RID`, `Callable`, and `Signal` are stored as strings ([GH-78517](https://github.com/godotengine/godot/pull/78517)).
+- Core: Fix scene load crash related to `_ready` ([GH-78654](https://github.com/godotengine/godot/pull/78654)).
+- Core: Ensure default node groups' call queue are processed ([GH-78713](https://github.com/godotengine/godot/pull/78713)).
+- Editor: Handle contextual editors gracefully when restoring layout ([GH-78611](https://github.com/godotengine/godot/pull/78611)).
+- Editor: Display a message about missing C# support on Android/iOS/Web platforms ([GH-78629](https://github.com/godotengine/godot/pull/78629)).
+- Editor: Improve script icon cache ([GH-78670](https://github.com/godotengine/godot/pull/78670)).
+- GDExtension: Fix text_server_adv compiling as a GDExtension ([GH-77532](https://github.com/godotengine/godot/pull/77532)).
+- GDExtension: Add GDExtension `@since` attribute ([GH-78518](https://github.com/godotengine/godot/pull/78518)).
+- GDScript: Fix a race condition in ScriptServer ([GH-76586](https://github.com/godotengine/godot/pull/76586)).
```

Comment:
Technically more of a core change, I relabeled the issue. But it doesn't matter much, can be left as is here.